animations/src/main/java/com/chiralbehaviors/CoRE/phantasm/model/PhantasmCRUD.java

```diff
@@ -868,147 +1014,76 @@ public RuleForm setSingularChild(RuleForm instance,
         return instance;
     }
 
-    private void checkAPPLY(NetworkAuthorization<RuleForm> facet,
-                            NetworkedModel<RuleForm, ?, ?, ?> networkedModel) {
-        if (!networkedModel.checkFacetCapability(model.getCurrentPrincipal()
-                                                      .getPrincipal(),
-                                                 facet, model.getKernel()
-                                                             .getAPPLY())) {
-            throw new SecurityException(String.format("%s does not have %s capability",
-                                                      model.getCurrentPrincipal(),
-                                                      model.getKernel()
-                                                           .getAPPLY()));
-        }
+    private boolean checkREAD(AttributeAuthorization<RuleForm, Network> stateAuth,
```

Comment:
Changed these to return boolean, rather than throwing exception, to comply with graphql semantics (i.e. silently fail and return appropriate value, null value (for gets) or emptyList (for list gets)